### PR TITLE
Create New-CsTeamsFilesPolicy.md

### PR DESCRIPTION
**Pull request by mayurkl100**

### CODE REVIEW

**Review by ss00dd — APPROVED** [review]

File: teams/teams-ps/teams/New-CsTeamsFilesPolicy.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+Required: False
+Position: Named
+Default value: False
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Description
+{{ Fill Description Description }}
```
Comment (michbrown-png) [inline teams/teams-ps/teams/New-CsTeamsFilesPolicy.md:113]: Please add text for the Description parameter

File: teams/teams-ps/teams/New-CsTeamsFilesPolicy.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Force
+{{ Fill Force Description }}
```
Comment (michbrown-png) [inline teams/teams-ps/teams/New-CsTeamsFilesPolicy.md:128]: Please add text for the force parameter

**Review by michbrown-png — CHANGES_REQUESTED** [review]

File: teams/teams-ps/teams/New-CsTeamsFilesPolicy.md (hunk 2 above)
Comment (mayurkl100) [reply teams/teams-ps/teams/New-CsTeamsFilesPolicy.md]: Removed this parameter as this is not supported with this cmdlet.

File: teams/teams-ps/teams/New-CsTeamsFilesPolicy.md (hunk 1 above)
Comment (mayurkl100) [reply teams/teams-ps/teams/New-CsTeamsFilesPolicy.md]: Removed this parameter as this is not supported with this cmdlet.

**Comment by michbrown-png** [comment]
@mayurkl100 , one last update needed

**Pull request merged**